openassessment/assessment/api/peer.py

```diff
@@ -862,6 +862,107 @@ def set_assessment_feedback(feedback_dict):
         raise PeerAssessmentInternalError(msg)
 
 
+def get_data_for_override_score(submission_uuid, student_item, rubric_dict):
+    """
+    Gets the data needed to render the score override section of the student data page.
+
+    Args:
+        submission_uuid (string): UUID of the student's submission
+        student_item (dict): The dictionary representation of a student item.
+        rubric_dict (dict): The dictionary representation of a rubric
+
+    Returns:
+        scores (dict): points_earned, pointed_possible, points_override
+    """
+    scores = {}
+    score_data = sub_api.get_latest_score_for_submission(submission_uuid)
+    if score_data:
+        scores['points_earned'] = score_data.get('points_earned', None)
+        scores['points_possible'] = score_data.get('points_possible', None)
+
+    # If there is no Score, retrieve points_possible from the rubric as there may not be any assessments yet.
+    if not scores.get('points_possible', None):
+        try:
+            rubric = rubric_from_dict(rubric_dict)
+            points_possible = rubric.points_possible
+        except InvalidRubric:
+            points_possible = None
+
+        scores['points_possible'] = points_possible
+    latest_override_score = sub_api.get_score_override(student_item)
+    if latest_override_score:
+        scores['points_override'] = latest_override_score['points_earned']
+
+    return scores
+
+
+def validate_override_data(points_override, points_possible):
```

Comment:
Can we mark this as private by prepending a `_` to the name? `_validate_override_data`